roles/bootstrap-os/tasks/bootstrap-archlinux.yml

```diff
@@ -0,0 +1,24 @@
+---
+#  raw: cat /etc/issue.net | grep '{{ bootstrap_versions }}'
+
+- name: Bootstrap | Check if bootstrap is needed
+  raw: which "{{ item }}"
+  register: need_bootstrap
+  failed_when: false
+  changed_when: false
+  with_items:
```

Comment:
Why run a loop over a single item?

Also, does Arch really not install Python by default?